pkg/api/login.go

```diff
@@ -427,10 +427,26 @@ func getLoginExternalError(err error) string {
 		return createTokenErr.ExternalErr
 	}
 
+	// unwrap until we get to the error message
 	gfErr := &errutil.Error{}
 	if errors.As(err, gfErr) {
-		return gfErr.Public().Message
+		return getFirstPublicErrorMessage(gfErr)
 	}
 
 	return err.Error()
 }
+
+// Get the first public error message from an error chain.
+func getFirstPublicErrorMessage(err *errutil.Error) string {
```

Comment:
outside of the scope of this PR: I wonder if it makes sense to add this to `errutil` itself.

Reply:
@sakjur would it make sense to bring a PR up adding this to errutil?